src/services/source/nativeIntegration.ts

```diff
@@ -9,6 +9,8 @@ import PostTransformationServiceSource from './postTransformation';
 import FetchHandler from '../../helpers/fetchHandlers';
 import tags from '../../v0/util/tags';
 import stats from '../../util/stats';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Why is this line added ? I'm seeing we are using CatchErr and FixMe in this file